gems/sorbet-runtime/test/types/builder_syntax.rb

```diff
@@ -302,6 +302,7 @@ def self.foo(x); end
           end
 
           it 'override default checked level to :tests, without checking tests' do
+            T::Private::RuntimeLevels._toggle_checking_tests(false)
```

Comment:
This ensures that even in pay-server, where we actually do toggle checking in tests, that it's off for the purpose of this test, to match the default behavior of our users.